### DISCUSSION TITLE
Difficulty split in song selection similar to stable.

### DISCUSSION THREAD

**Discussion by Lunethia** [ppy/osu#21820] [Ideas] [answered]
Hello, I have never done this before, so I am unfamiliar with style stuffs, sorry ; - ;
I watched all of peppy's interviews and he always said to make a github issue whenever there's something one would inquire- So here I am!
I tried having a look through active issues and noticed that it had been brought up before, but wasn't being discussed, if this has already been resolved/decided on, then I apologize for wasting your time.

Anyway! 

In stable you can do this:

![Github Osu Difficulty 1](https://user-images.githubusercontent.com/52287610/209711002-4521d10d-4f06-4a18-aec2-05d256fa2dd5.png)

By selecting "No Grouping" and "Difficulty" all currently installed beatmapsets are split into their difficulties and reshuffled in a way that would make them go in order by difficulty.

This is not the case in Lazer and its really confusing, it doesn't seem to work at all.

Is this already possible and I'm being stupid? If not, will this be added? If not, can this be added by me somehow? It's really helpful to see all the difficulties of the same star rating listed right next to each other rather than seeing the entire sets and having to look for which difficulty is one that would even be playable for me, or one I want to play right now.
Being used to pretty much always using this search method in stable its a bit sad to see its seemingly impossible in lazer!

Anyway, thanks for reading!
Have a nice day! :)

**Comment by bdach** [comment, marked as answer]
Yet to be added.

https://github.com/ppy/osu/issues/4919

**Reply by Lunethia** [reply]
Oh, yes, sorry for the bump. That is the one I already saw beforehand.
